config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics.yaml

```diff
@@ -99,3 +99,38 @@ periodics:
         privileged: true
   annotations:
     testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
+- name: periodic-cluster-api-provider-azure-e2e-full
+  decorate: true
+  decoration_config:
+    timeout: 4h
+  interval: 12h
```

Comment:
made 12 hours, but we can reduce
do we know how long take to run the full?

Reply:
https://prow.k8s.io/job-history/gs/kubernetes-jenkins/pr-logs/directory/pull-cluster-api-provider-azure-e2e-full looks like it takes around 1h so 2h timeout should be enough

Reply:
update the timeout to 2h